oregon/static/titlecities.txt

```diff
@@ -0,0 +1,62 @@
+label,lat,lon,url
```

Comment:
I think this works for now; can revisit how the file is generated at a later date.